docroot/modules/custom/sitenow_people/sitenow_people.module

```diff
@@ -26,6 +26,29 @@ use Drupal\views\Entity\View;
 use Drupal\views\ViewExecutable;
 use Drupal\views\Views;
 
+/**
+ * Unsets a block from the given layout builder parts.
+ *
+ * @param array &$variables
+ *   The variables array.
+ * @param array $layout_parts
+ *   The layout parts to loop through.
+ * @param string $block_id
+ *   The block ID to unset.
+ */
+function sitenow_people_unset_block(array &$variables, array $layout_parts, string $block_id) {
+  foreach ($layout_parts as $layout_part) {
+    if (isset($variables['content']['_layout_builder'][2][$layout_part])) {
```

Comment:
I'm a little worried about the fragility of this check. If we were to add another section ahead of this (or move the content moderation section into the header) this would break, I think.

Reply:
I agree it is fragile.  We are doing something similar here too: https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/sitenow_articles/sitenow_articles.module#L307.  Any suggestions on how to do this better?

Reply:
Discussed in Slack with @bspeare and we are going to try an approach that loops through all LB levels and looks for matches on the `$block_id`. This will need to be updated once we have section UUID's but other than that should not produce errors for most situations we can think of.

Reply:
@pyrello I updated the check to loop through all the layouts.  Can you check it out https://github.com/uiowa/uiowa/pull/6570/commits/ce56362140f8fe8673d77dfd6043d3e336ef1682?